zokrates_parser/src/zokrates.pest

```diff
@@ -53,11 +53,14 @@ _mut = {"mut"}
 
 // Statements
 statement = { (iteration_statement // does not require semicolon
-              | ((return_statement
+              | ((log_statement
+                |return_statement
                 | definition_statement
                 | assertion_statement
                 ) ~ semicolon)) ~ NEWLINE* }
 
+log_statement = { "log" ~ "(" ~ "\"" ~ format_string ~ "\"" ~ "," ~ expression_list ~ ")"}
+format_string = { (!"\"" ~ ANY)* }
```

Comment:
we already have `string` defined as this above, maybe `log_statement` should share the same defs as `assertion_statement`, because the one used there supports escaped quoutes iirc